common/djangoapps/student/models.py

```diff
@@ -2066,10 +2066,14 @@ def add_to_profile_url(self, course_key, course_name, cert_mode, cert_url, sourc
 
     def _cert_name(self, course_name, cert_mode):
         """Name of the certification, for display on LinkedIn. """
-        return self.MODE_TO_CERT_NAME.get(
```

Comment:
I had to track down where `MODE_TO_CERT_NAME` was defined to see how it looked -- it's located on [line 2007](https://github.com/edx/edx-platform/pull/14031/files#diff-e2ffffdf6d67c9e0dee5bc10a67eb8bfL2007).  I did a search on the code base and it looks like `_cert_name()` appears to be the only reference.  It would be great if we can consolidate or at least move it closer to the place it's being used.